Automattic/jetpack
# Widget: remove Pay with PayPal widget from Legacy Widget block

#### Summary
The Pay with PayPal widget is not functioning properly inside of the Legacy Widget block. Since this functionality is available with the Pay with PayPal block we should hide the widget in favor of the block.

Fixes Automattic/wp-calypso#57838

#### Changes proposed in this Pull Request:
* Add widget to the `widget_types_to_hide_from_legacy_widget_block` filter.

#### Jetpack product discussion
#21069

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:
* Make sure you have a paid Jetpack plan to see the widget
* Go to Appearance ⇢ Customizer ⇢ Widgets
* Add the Legacy Widget block and check the drop-down for `Pay with PayPal`.

#### Screenshots
<img width="792" alt="Markup 2021-11-30 at 18 54 25" src="https://user-images.githubusercontent.com/33258733/144102614-76179bd5-1213-43a6-bd16-5b2d92db9303.png">

<img width="594" alt="Markup 2021-11-30 at 18 54 11" src="https://user-images.githubusercontent.com/33258733/144102669-5c0603a4-8241-436b-b7e8-2e649e99e96d.png">
